src/content/toys.ts

```diff
@@ -32,17 +32,88 @@ function makeBlock() {
     overlaydiv.style.position = "fixed"
     overlaydiv.style.display = "block"
     overlaydiv.style.width = String(window.innerWidth)
-    overlaydiv.style.height = String(window.innerHeight)
-    overlaydiv.style.top = "0"
-    overlaydiv.style.left = "0"
-    overlaydiv.style.right = "0"
-    overlaydiv.style.bottom = "0"
-    overlaydiv.style.zIndex = "1000"
-    overlaydiv.style.opacity = "0.5"
+    overlaydiv.style.height = String(document.documentElement.scrollHeight)
+    overlaydiv.style.top = "0px"
+    overlaydiv.style.bottom = "0px"
+    overlaydiv.style.left = "0px"
+    overlaydiv.style.right = "0px"
+    overlaydiv.style.zIndex = "100"
+    overlaydiv.style.opacity = "0.8"
     document.body.appendChild(overlaydiv)
     return overlaydiv
 }
 
+export function drawable() {
+    make_drawable(makeBlock())
+}
+
+const clickX = []
+const clickY = []
+const clickDrag = []
+let ink
+
+let eraser = false;
+export function eraser_toggle() {
+    eraser = !eraser
+}
+
+function addClick(x, y, dragging) {
+    clickX.push(x)
+    clickY.push(y)
+    clickDrag.push(dragging)
+}
+
+function redraw(context) {
+    if(eraser) {
+        context.globalCompositeOperation = "destination-out"
+        context.lineWidth = 18
+    } else {
+        context.globalCompositeOperation = "source-over"
+        context.lineWidth = 3
+    }
+    context.strokeStyle = "#000000"
+    context.lineJoin = "round"
+    for(let i=0; i < clickX.length; i++) {
+        context.beginPath()
+        if(clickDrag[i] && i){
+            context.moveTo(clickX[i-1], clickY[i-1])
+        } else {
+            context.moveTo(clickX[i]-1, clickY[i])
+        }
+        context.lineTo(clickX[i], clickY[i])
+        context.closePath()
+        context.stroke()
+    }
+}
+
+function make_drawable(overlaydiv) {
+    overlaydiv.style.position = "absolute"
+    const c = document.createElement("canvas")
+    overlaydiv.appendChild(c)
+    const context = c.getContext("2d")
+    // making the canvas full screen
+    c.height = document.documentElement.scrollHeight
+    c.width = window.innerWidth*0.98
```

Comment:
canvas overflows if the 0.98 is not added. I do not know how to fix this

Reply:
Should we care? It seems to work fine with the magic `*0.98`. I'm happy for weird workarounds to stay in the code - just add a comment saying what it does